doc/source/new-project-guide/level-5-tutorial.rst

```diff
@@ -218,11 +218,17 @@ We require that each PR includes a news item as a ``<branch-name>.rst`` file und
         git checkout skpkg-migration
         git pull origin skpkg-migration
 
-#. Make a copy of ``news/TEMPLATE.rst`` and rename it to ``news/<branch-name>.rst``.
+#. Make a copy of ``news/TEMPLATE.rst`` and rename to ``news/<branch-name>.rst``. We receommend use the following command to streamline this process.
 
     .. code-block:: bash
 
-        cp news/TEMPLATE.rst news/skpkg-migration.rst
+        cp news/TEMPLATE.rst news/$(git rev-parse --abbrev-ref HEAD).rst
+
+#. (optional) we recommend that you setup ``alias`` for the above command so that you can automate this news file creation process by just running ``cpnews`` command. You can add the following line to your ``~/.bashrc`` or ``~/.zshrc`` file and run ``source ~/.bashrc`` or ``source ~/.zshrc`` to apply the changes.
```

Comment:
Here we can say something like "(optional) if you are using a linux shell you can set up an ``alias`` to make the creation of the news file ready for editing much quicker and easier....."

Reply:
Done - as recommended.